src/plugin.validation.dns.tencent/Tencent.cs

```diff
@@ -0,0 +1,186 @@
+            //删除
+            DelRecord(domain, subDomain);
+            //增加
+            var mod = "dnspod";
+            var ver = "2021-03-23";
+            var act = "CreateRecord";
+            var region = "";
+            var endpoint = "dnspod.tencentcloudapi.com";
+            var hpf = new HttpProfile
+            {
+                ReqMethod = "POST",
+                Endpoint = endpoint,
+            };
+            var cpf = new ClientProfile(ClientProfile.SIGN_TC3SHA256, hpf);
+            var client = new CommonClient(mod, ver, _cred, region, cpf);
+            var param = new
+            {
+                Domain = domain,
+                SubDomain = subDomain,
+                RecordType = "TXT",
+                RecordLine = "默认",
+                Value = value,
+            };
+            var req = new CommonRequest(param);
+            var resp = client.Call(req, act);
+            //Console.WriteLine(resp);
+            return true;
+        }
+
+        /// <summary>
+        /// 删除解析记录
+        /// </summary>
+        /// <param name="domain">域名</param>
+        /// <param name="subDomain">主记录</param>
+        /// <returns></returns>
+        private bool DelRecord(string domain, string subDomain)
+        {
+            subDomain = subDomain.Replace($".{domain}", "");
+            //检查
+            var recordId = GetRecordID(domain, subDomain);
+            if (recordId == default) return false;
+            //删除
+            var mod = "dnspod";
+            var ver = "2021-03-23";
+            var act = "DeleteRecord";
+            var region = "";
+            var endpoint = "dnspod.tencentcloudapi.com";
+            var hpf = new HttpProfile
+            {
+                ReqMethod = "POST",
+                Endpoint = endpoint,
+            };
+            var cpf = new ClientProfile(ClientProfile.SIGN_TC3SHA256, hpf);
+            var client = new CommonClient(mod, ver, _cred, region, cpf);
+            var param = new { Domain = domain, RecordId = recordId };
+            var req = new CommonRequest(param);
+            var resp = client.Call(req, act);
+            //Console.WriteLine(resp);
+            return true;
+        }
+
+        /// <summary>
+        /// 获取解析ID
+        /// </summary>
+        /// <param name="domain">域名</param>
+        /// <param name="subDomain">主记录</param>
+        /// <returns></returns>
+        private long GetRecordID(string domain, string subDomain)
+        {
+            var mod = "dnspod";
+            var ver = "2021-03-23";
+            var act = "DescribeRecordList";
+            var region = "";
+            var endpoint = "dnspod.tencentcloudapi.com";
+            var hpf = new HttpProfile
+            {
+                ReqMethod = "POST",
+                Endpoint = endpoint,
+            };
+            var cpf = new ClientProfile(ClientProfile.SIGN_TC3SHA256, hpf);
+            var client = new CommonClient(mod, ver, _cred, region, cpf);
```

Comment:
I see this code repeated 3 times, consider consolidating this into a helper function